backend/src/model/entity/plants_impl.rs

```diff
@@ -14,6 +15,42 @@ use crate::{
 use super::Plants;
 
 impl Plants {
+    /// Get the top 10 plants matching the search query.
```

Comment:
Please write in an issue which features are missing.

Reply:
See #379